stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -336,28 +344,60 @@ func.func @batch_norm_grad_bounds(
 
 // -----
 
-// CHECK-LABEL: func @batch_norm_train_bounds
-func.func @batch_norm_train_bounds(
-  %input: tensor<2x?xf32, #stablehlo.bounds<?, 64>>,
```

Comment:
It seems we have deleted this?

Reply:
Thanks, added back in BatchNormTraining PR